Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: all run build
```

Comment:
I usually like to organize my makefiles like this:
```
.PHONY: all
all:
    command

.PHONY: run
run:
    command

etc...
```

This is personal preference and not semantically changing anything, so I'll merge this regardless, but thought I'ld share my preferences ^^